src/platform/datapath_raw_socket.c

```diff
@@ -278,8 +289,7 @@ CxPlatResolveRoute(
     CXPLAT_DBG_ASSERT(!QuicAddrIsWildCard(&Route->RemoteAddress));
 
     //
-    // Find the best next hop IP address. If not cached, this may result in (and
-    // block on) network IO.
+    // Find the best next hop IP address.
```

Comment:
Can't `GetBestRoute2` block on IO too? If so, it needs to be on the other thread too.

Reply:
It does not block on IO.